.circleci/config.yml

```diff
@@ -7,110 +9,56 @@ jobs:
-    - run:
-        name: Install Kakadu
-        command: |
-          if [ ! -d "kakadu" ]; then
-            mkdir ~/downloads
-            wget http://kakadusoftware.com/wp-content/uploads/2014/06/KDU77_Demo_Apps_for_Linux-x86-64_150710.zip -O ~/downloads/kakadu.zip
-            unzip ~/downloads/kakadu.zip
-            mv KDU77_Demo_Apps_for_Linux-x86-64_150710 kakadu
-          fi
-          sudo cp kakadu/*.so /usr/lib
-          sudo cp kakadu/* /usr/bin
-
-    - restore_cache:
-        keys:
-        - v1-bundle-{{ checksum "Gemfile" }}--{{ checksum "<< parameters.project >>.gemspec" }}-<< parameters.ruby_version >>
-
-    - run:
-        name: Update bundler
-        command: |
-          echo 'export BUNDLER_VERSION=<< parameters.bundler_version >>' >> $BASH_ENV
-          gem install bundler -v << parameters.bundler_version >>
-
-    - run:
-        name: Install dependencies
-        command: bundle check || bundle install
+      - samvera/cached_checkout
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt-get update
+            sudo apt-get install ghostscript ufraw-batch libpng-dev imagemagick ffmpeg libreoffice
+      - run:
+          name: Install Kakadu
+          command: |
+            if [ ! -d "kakadu" ]; then
+              mkdir ~/downloads
+              wget http://kakadusoftware.com/wp-content/uploads/2014/06/KDU77_Demo_Apps_for_Linux-x86-64_150710.zip -O ~/downloads/kakadu.zip
+              unzip ~/downloads/kakadu.zip
+              mv KDU77_Demo_Apps_for_Linux-x86-64_150710 kakadu
+            fi
+            sudo cp kakadu/*.so /usr/lib
+            sudo cp kakadu/* /usr/bin
 
-    - save_cache:
-        key: v1-bundle-{{ checksum "Gemfile" }}--{{ checksum "<< parameters.project >>.gemspec" }}-<< parameters.ruby_version >>
-        paths:
-        - ~/project/vendor/bundle
+      - samvera/bundle_for_gem:
+          ruby_version: << parameters.ruby_version >>
+          bundler_version: << parameters.bundler_version >>
+          project: << parameters.project >>
 
-    - run:
-        name: Load config into solr
-        command: |
-          cd solr/config
-          zip -1 -r solr_config.zip ./*
-          curl -H "Content-type:application/octet-stream" --data-binary @solr_config.zip "http://localhost:8985/solr/admin/configs?action=UPLOAD&name=solrconfig"
-          curl -H 'Content-type: application/json' http://localhost:8985/api/collections/ -d '{create: {name: hydra-test, config: solrconfig, numShards: 1}}'
+      - samvera/install_solr_core
 
-    - run:
-        name: Call Rubocop
-        command: bundle exec rubocop
+      - samvera/parallel_rspec
 
-    - run:
-        name: Run rspec in parallel
-        command: |
-          mkdir /tmp/test-results
-          bundle exec rspec $(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)
+      - run:
```

Comment:
Should rubocop come before the rspec so it can fail quickly on style issues (like it was before) or was this deliberate so you can see all results (rspec and rubocop) from a ci run?